### DISCUSSION TITLE
How can i use this repo?

### DISCUSSION THREAD

**Discussion by CODGhost911** [Saisha16/DSA-Source-Codes#7] [Q&A] [answered]
Tips plzz😁

**Comment by Atharvkote** [comment, marked as answer]
## Well, you were the who wanted learn DSA In Python Right?
This Repo is None ur Businesses find a Repo fro DSA in Python this repo is for Java Devs. Or C Devs.
You can perfer Code With harrpa 100 days python coure available on youtube for freee!!!

I had gone thourgh  a lot to take ,I bet you will enjoy it😄😊

**Reply by CODGhost911** [reply]
I that kinda good reply mate 👌
